packs/pagerduty/actions/ack_incident.py

```diff
@@ -0,0 +1,16 @@
+import json
```

Comment:
This import seems to be unused. Same goes with requests.

(applies to all python files)